models/sharky/solana/sharky_solana_events.sql

```diff
@@ -1,4 +1,5 @@
 {{ config(
+    tags=['dunesql'],
     alias = alias('events'),
     partition_by = ['block_date'],
```

Comment:
could you also increase partition size to monthly and apply a prehook, like here?
https://github.com/duneanalytics/spellbook/pull/3707/files#diff-0576be6634b68cdb81185a4376f442e9be34f967a6f182656a63b89c87a1f3d0R6-R10

we're using that prehook on larger data volume spells on trino (soon will be universally built at project level so we don't have to config each model moving forward). as for the partition, we're looking to increase universally. if `block_date` is used at all on the dune app for queries, you can add a new column to be safe, otherwise feel free to override existing

Reply:
Yeah `block_date` is used so I'll add a new one